From 01f6654ad2be6b9bbf550c464e4d303c0b742185 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sat, 17 Jan 2026 16:01:52 +0100
Subject: [PATCH] Add missing test files to EXTRA_DIST
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add EXTRA_DIST entries for the tests introduced in the "tests: add test
cases for import without uid" patch.

Signed-off-by: Michał Górny <mgorny@gentoo.org>
---
 tests/openpgp/Makefile.am | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/openpgp/Makefile.am b/tests/openpgp/Makefile.am
index bf2f11a08..0a884203a 100644
--- a/tests/openpgp/Makefile.am
+++ b/tests/openpgp/Makefile.am
@@ -164,6 +164,11 @@ TEST_FILES = pubring.asc secring.asc plain-1o.asc plain-2o.asc plain-3o.asc \
 	     tofu/cross-sigs/871C2247-3.txt \
 	     tofu/cross-sigs/871C2247-4.gpg \
 	     tofu/cross-sigs/README \
+	     import-incomplete/primary+revocation.asc \
+	     import-incomplete/primary+subkey+sub-revocation.asc \
+	     import-incomplete/primary+subkey+sub-sig.asc \
+	     import-incomplete/primary+uid.asc \
+	     import-incomplete/primary+uid-sig.asc \
 	     key-selection/0.asc \
 	     key-selection/1.asc \
 	     key-selection/2.asc \
